### PR TITLE
[ACHIEVEMENT] Mr. Miyagi

### PR DESCRIPTION
**Pull request by Thatkookooguy**
# Change Summary

 - create `Mr. Miyagi` achievement for getting tests to 100% [ relates to #13 ]
 - add tests for reactionsOnEveryComment + some fixes to the achievement itself

If you have a different reference that suits this scenario better, let me know :-)

### CODE REVIEW

**Comment by Thatkookooguy** [comment]
@ortichon make sure you go over #72 , so I can merge it before you review this one (and tests will pass once more!)

File: achievements/mrMiyagi.achievement.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        ].join(''),
+        relatedPullRequest: pullRequest.id
+      };
+
+      shall.grant(pullRequest.creator.username, achievement);
+    }
+  }
+};
+
+function coverageTurned100(pullRequest) {
```
Comment (ortichon) [inline achievements/mrMiyagi.achievement.js]: consider changing this function name to something like:
`isCoverageTurened100` to keep the convention for functions that return `boolean`

**Comment by ortichon** [comment]
@Thatkookooguy sorry for the nit-pick

**Comment by ortichon** [comment]
@Thatkookooguy 
ohh, and please update this branch with master

**Comment by Thatkookooguy** [comment]
@ortichon made the changes

**Pull request merged**